#uint64 timestamp			# output timestamp in us since system boot
uint8 ninputs				# number of valid inputs
uint16[8] input    # input rpm command
